src/main/java/at/hannibal2/skyhanni/features/rift/area/dreadfarm/RiftAgaricusCap.kt

```diff
@@ -36,19 +37,19 @@ class RiftAgaricusCap {
         when (currentLocation.getBlockStateAt().toString()) {
             "minecraft:brown_mushroom" -> {
                 return if (location != currentLocation) {
-                    startTime = System.currentTimeMillis()
+                    startTime = SimpleTimeMark.now()
                     currentLocation
                 } else {
-                    if (startTime == -1L) {
-                        startTime = System.currentTimeMillis()
+                    if (startTime.isFarFuture()) {
```

Comment:
this has now changed logic. before we were using 0 and -1 differently, maybe use nullable instead?

Reply:
here im using farFuture to represent -1, this worked from my testing of the feature

Reply:
we also never checked if its 0 or set it back to 0 so maybe I could just make them all far past?

Reply:
since you have tested it, its fine